lib/swaps/SwapClientManager.ts

```diff
@@ -7,8 +7,16 @@ import Logger, { Loggers } from '../Logger';
 import Pool from '../p2p/Pool';
 import { errors } from './errors';
 import { Currency } from '../orderbook/types';
-import { isLndClient, isRaidenClient } from './types';
 import { Models } from '../db/DB';
+import { SwapClientType } from '../constants/enums';
+
+function isRaidenClient(swapClient: SwapClient): swapClient is RaidenClient {
```

Comment:
Minor nit: do we want to keep `isRaiden`/`isLndClient` here or move them back to `types`?

Reply:
After I changed the method to use a type guard (e.g. `swapClient is RaidenClient`) I was running into circular dependency errors between `SwapClientType` and `SwapClient`, so I reorganized it like this. It seems fine since at least for now since we're not using it anywhere else.